src/SerializeResult.cc

```diff
@@ -150,9 +153,10 @@ sos::Object WrapParseResultRefract(snowcrash::ParseResult<snowcrash::Blueprint>&
 }
 
 sos::Object drafter::WrapResult(snowcrash::ParseResult<snowcrash::Blueprint>& blueprint,
-                                const WrapperOptions& options)
+                                const WrapperOptions& options,
+                                ConversionContext& context)
```

Comment:
Yes, but I think that should be a part of "cleaning public api" which
should take care of it without doing context building in so many places as
we have in this PR.

On Wed, Apr 13, 2016, 17:02 klokane <notifications@github.com> wrote:

> In src/SerializeResult.cc
> <https://github.com/apiaryio/drafter/pull/295#discussion_r59564636>:
>
> > @@ -150,9 +153,10 @@ sos::Object WrapParseResultRefract(snowcrash::ParseResult<snowcrash::Blueprint>&
> >  }
> >
> >  sos::Object drafter::WrapResult(snowcrash::ParseResult<snowcrash::Blueprint>& blueprint,
> > -                                const WrapperOptions& options)
> > +                                const WrapperOptions& options,
> > +                                ConversionContext& context)
>
> In curent time it seems, but we will need merge, but we will need to merge
> Errors/Warnings at least there
> https://github.com/apiaryio/drafter/blob/master/src/main.cc#L82
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/apiaryio/drafter/pull/295/files/d83d56b46deb6e84181a8e3b5cccb22eca819203#r59564636>
>